lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -90,6 +92,27 @@ class DetailsRenderer {
   }
 
   /**
+   * @param {!DetailsRenderer.LinkDetailsJSON} details
+   * @return {!Element}
+   */
+  _renderLink(details) {
```

Comment:
let's call `_renderURL` `_renderURLText` instead to clarify it's never linkified.

Reply:
> let's call _renderURL _renderURLText instead to clarify it's never linkified.

done